packages/mason_cli/lib/src/commands/make.dart

```diff
@@ -232,6 +267,56 @@ class _MakeCommand extends MasonCommand {
       return value;
     }
   }
+
+  /// Starts watching for changes.
+  ///
+  /// Should only be called when the flag `--watch` is specified.
+  ///
+  /// When watching, any changes done within the brick's directory will trigger
+  /// a new make command [run] (with the same arguments as the first [run]).
+  ///
+  /// This method does nothing when:
+  /// - The [_brick] does not have a path.
+  /// - The command is already watching for changes.
+  ///
+  /// See also:
+  ///
+  /// * [DirectoryWatcher], watcher used to trigger new makes.
+  Future<void> _watch() async {
+    if (_brick.path == null || _isWatching) {
```

Comment:
@felangel when would a BrickYaml not have a path (i.e. when can _brick.path be null)?